### PR TITLE
ITE MAINTAINERS: add section for ITE

### PR DESCRIPTION
**Pull request by RuibinChang**
Add section for ITE.

Signed-off-by: Ruibin Chang <ruibin.chang@ite.com.tw>

### CODE REVIEW

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -1410,6 +1410,22 @@ Espressif Platforms:
...
+        - RuibinChang
+        - DinoLi
+        - GTLin08
+        - jackrosenthal
+        - Keith Short
+        - brockus-zephyr
+        - Yuval Peress
+        - sjg20
+    files:
+        - dts/riscv/
```
Comment (nashif) [inline MAINTAINERS.yml:1425]: need to be more specific about the files here here, this covers all riscv but non of the ITE related platforms, SoCs.

**Review by nashif — CHANGES_REQUESTED** [review]

File: MAINTAINERS.yml (hunk 1 above)
Comment (RuibinChang) [reply MAINTAINERS.yml]: Done

**Review by nashif — APPROVED** [review]

**Review by keith-zephyr — APPROVED** [review]

**Review by brockus-zephyr — APPROVED** [review]

**Comment by nashif** [comment]
btw, any reason why we do not have a maintainer?

**Comment by RuibinChang** [comment]
> btw, any reason why we do not have a maintainer?

Done

**Pull request merged**